dummy

```diff
@@ -0,0 +1 @@
+
```

Comment:
You can remove it, it only existed because i'm a bit new to some git
features...

On Tue, Jun 14, 2016 at 8:41 PM, Vassil Vassilev <notifications@github.com>
wrote:

> In dummy
> <https://github.com/vgvassilev/creduce/pull/1#discussion_r67019380>:
>
> > @@ -0,0 +1 @@
> > +
>
> Is this necessary or I can remove it when merging?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vgvassilev/creduce/pull/1/files/9541f774d8ae5fc8aa1b524622eae61e7d216c37#r67019380>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASnDj2eSvlm1o5tS6xS5IFCA3uXfaa-rks5qLuexgaJpZM4I1kgl>
> .
>